README.md

```diff
@@ -23,71 +26,81 @@ Also, check the documentation of Wrangler Transform [here](transform/docs/Wrangl
 * Recipes
   * [Log Parsing](demos/Parsing-Apache-Log.md)
   * [HL7 CCDA XML Parsing](demos/HL7-CCDA-XML-Parsing.md)
-  * [CSV Parsing and micro-wrangling columns](demos/CSV-Parsing-And-Extraction.md)
+  * [CSV Parsing and Extracting Column Values](demos/CSV-Parsing-And-Extraction.md)
+
 
-  
 ## Concepts
 
-This implementation of wrangler defines the following concepts. Please familiarize yourself with these concepts. 
+This implementation of Wrangler uses these concepts.
 
 ### Record
 
-A Record is a collection of field names and field values. 
+A *Record* is a collection of field names and field values.
 
 ### Column
 
-A Column is a data value of any supported java type, one for each Record.
+A *Column* is a data value of any of the supported Java types, one for each record.
 
 ### Directive
 
-A Directive is a single data manipulation instruction specified to either transform, filter or pivot a single record into zero or more records. A directive can generate one or more Steps to be executed by the Pipeline. 
+A *Directive* is a single data manipulation instruction, specified to either transform,
+filter, or pivot a single record into zero or more records. A directive can generate one
+or more *steps* to be executed by a Pipeline.
```

Comment:
I think there are also directives that don't do anything in a pipeline. I don't know if they are in this release or not. @nitinmotgi ?